lib/examine-package.js

```diff
@@ -180,9 +238,75 @@ function hasNpmLintScript(json) {
     }
 }
 
+function shouldUseBabelCli(json) {
+    if (!json.devDependencies['babel-cli']) {
```

Comment:
Uncessary with multiple if statements. Use if else, or just an if and a default return value. Same goes for the rest of the tests below.